data/books.json

```diff
@@ -1,6 +1,6 @@
 [
```

Comment:
![Screenshot from 2023-05-25 03-42-30](https://github.com/shafiuyushawu/school-liabrary/assets/93039359/2efc3411-fd35-43fa-a8fb-1bb4dcf84a12)

- [ ] It would be nice if you fix the Live Demo, Run tests and Deployment link which you can see in the image above ⬆️ because they are broken 😨 Kindly fix them or remove them if you don't have any section for these links to make your readme descriptive and professional 👍